### ISSUE TITLE
Feat: Update Footer Layout for Social Media Icons 

### ISSUE THREAD

**Issue by Amruta7203** [subhadeeproy3902/BloxAI#137]
### Describe the feature

The social media icons in the footer are currently positioned in a corner, making them less noticeable to users. This reduces their effectiveness in driving visitors to your social media profiles.
I propose to change the placement of the social media icons in the footer to improve their visibility and encourage user engagement.

### Add ScreenShots

![image](https://github.com/subhadeeproy3902/BloxAI/assets/116087736/a5293652-06f7-4f4e-a94f-ecda8d3cc2dd)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by subhadeeproy3902** [comment]
@Amruta7203 I dont think so its required cuz already they are given in link format in the format... icona were for show... and the footer it kinda looking sleak. and awesome so... i dont want it to be changed... Thank you
